pokedex/src/main/java/hive/pokedex/exception/EntityAlreadyExistsException.java

```diff
@@ -0,0 +1,8 @@
+package hive.pokedex.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.CONFLICT, reason = "Entity already registered, try again")
```

Comment:
```suggestion
@ResponseStatus(code = HttpStatus.CONFLICT, reason = "Entity already registered")
```
"try again" da impressão de tentar novamente com os mesmos dados